### PR TITLE
Adding helm chart for admission controller

### PR DESCRIPTION
**Pull request by amitschendel**

### CODE REVIEW

**Comment by amitschendel** [comment]
Merge when https://github.com/kubescape/operator/pull/226 is merged.

**Comment by amitschendel** [comment]
And bump operator version once operator is released.

**Comment by matthyx** [comment]
> And bump operator version once operator is released.

converted to draft until then

File: charts/kubescape-operator/templates/operator/clusterrole.yaml [hunk 1]
```diff
@@ -19,6 +19,9 @@ rules:
   - apiGroups: ["spdx.softwarecomposition.kubescape.io"]
     resources: ["vulnerabilitymanifests", "vulnerabilitymanifestsummaries", "workloadconfigurationscans", "workloadconfigurationscansummaries", "openvulnerabilityexchangecontainers", "sbomsyftfiltereds", "sbomsyfts"]
     verbs: ["get", "watch", "list", "delete"]
+  - apiGroups: ["kubescape.io"]
+    resources: ["runtimerulealertbindings"]
+    verbs: ["list", "watch"]
```
Comment (matthyx) [inline charts/kubescape-operator/templates/operator/clusterrole.yaml:24]: can you add get here for future proofing?

**Review by matthyx — COMMENTED** [review]
just a nit for RBAC

**Review by matthyx — APPROVED** [review]

**Pull request merged**